Tools/CISupport/ews-build/steps_unittest.py

```diff
@@ -5560,6 +5560,19 @@ def test_success_pr_validators_case(self):
         self.assertEqual(self.getProperty('reviewers_full_names'), ['WebKit Reviewer'])
         return rc
 
+    def test_success_pr_validators_not_reviewer(self):
+        self.setupStep(ValidateCommitterAndReviewer())
+        ValidateCommitterAndReviewer.get_reviewers = lambda x, pull_request, repository_url=None: ['geoffreygaren']
```

Comment:
From https://webkit.org/team/ it seems like Geoffrey Garen is a WebKit reviewer

Reply:
This is confusing, but `geoffreygaren` isn't a reviewer in unit tests, which is why this works.